Multiple tests in ClassificationIT are failing

**Build scan**: https://gradle-enterprise.elastic.co/s/73luwxfnclwxs

**Repro line**:

```
./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testWithOnlyTrainingRowsAndTrainingPercentIsFifty_DependentVariableIsBoolean" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testSingleNumericFeatureAndMixedTrainingAndNonTrainingRows" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testUpdateAnalytics" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testSetUpgradeMode_ExistingTaskGetsUnassigned" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testTwoJobsWithSameRandomizeSeedUseSameTrainingSet" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testDependentVariableCardinalityTooHighButWithQueryMakesItWithinRange" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testDependentVariableIsAliasToNested" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testWithOnlyTrainingRowsAndTrainingPercentIsHundred" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testWithOnlyTrainingRowsAndTrainingPercentIsFifty_DependentVariableIsDouble" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8

./gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ClassificationIT.testWithOnlyTrainingRowsAndTrainingPercentIsFifty_DependentVariableIsKeyword" \
  -Dtests.seed=1C06ACF456D7250A \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SY \
  -Dtests.timezone=Europe/Volgograd \
  -Druntime.java=8
```

**Reproduces locally?**: no

**Applicable branches**: This specific failure happened on `7.9` but build history shows failures on `master` as well.

**Failure history**:

We see [155 failure documents](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-1M,mode:relative,to:now))&_a=(columns:!(_source),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'class:%22org.elasticsearch.xpack.ml.integration.ClassificationIT%22'),sort:!(time,desc))) in build stats for the prior month.

**Failure excerpt**:

Looking at the output this is probably a test setup issue rather than a specific test issue:

```
10:00:37     java.lang.AssertionError: Stats were: {"id":"classification_training_percent_is_50_boolean","state":"starting","progress":[{"phase":"reindexing","progress_percent":1},{"phase":"loading_data","progress_percent":0},{"phase":"feature_selection","progress_percent":0},{"phase":"coarse_parameter_search","progress_percent":0},{"phase":"fine_tuning_parameters","progress_percent":0},{"phase":"final_training","progress_percent":0},{"phase":"writing_results","progress_percent":0},{"phase":"inference","progress_percent":0}],"data_counts":{"training_docs_count":0,"test_docs_count":0,"skipped_docs_count":0},"memory_usage":{"peak_usage_bytes":0,"status":"ok"},"node":{"id":"3JoP5p90RJugcI1l-tjB2w","name":"integTest-2","ephemeral_id":"JGR_vyedRuavrBcMaERnuw","transport_address":"127.0.0.1:38155","attributes":{"testattr":"test","ml.machine_memory":"101375320064","ml.max_open_jobs":"20","xpack.installed":"true","transform.node":"true"}},"assignment_explanation":""}
10:00:37     Expected: <stopped>
10:00:37          but: was <starting>
10:00:37         at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
10:00:37         at org.junit.Assert.assertThat(Assert.java:956)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.assertIsStopped(MlNativeDataFrameAnalyticsIntegTestCase.java:207)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.lambda$waitUntilAnalyticsIsStopped$0(MlNativeDataFrameAnalyticsIntegTestCase.java:157)
10:00:37         at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:951)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.waitUntilAnalyticsIsStopped(MlNativeDataFrameAnalyticsIntegTestCase.java:157)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.waitUntilAnalyticsIsStopped(MlNativeDataFrameAnalyticsIntegTestCase.java:153)
10:00:37         at org.elasticsearch.xpack.ml.integration.ClassificationIT.testWithOnlyTrainingRowsAndTrainingPercentIsFifty(ClassificationIT.java:275)
10:00:37         at org.elasticsearch.xpack.ml.integration.ClassificationIT.testWithOnlyTrainingRowsAndTrainingPercentIsFifty_DependentVariableIsBoolean(ClassificationIT.java:354)
10:00:37 
10:00:37     java.lang.RuntimeException: Had to resort to force-stopping jobs, something went wrong?
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.stopAnalyticsAndForceStopOnError(MlNativeDataFrameAnalyticsIntegTestCase.java:114)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.cleanUpAnalytics(MlNativeDataFrameAnalyticsIntegTestCase.java:91)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeDataFrameAnalyticsIntegTestCase.cleanUpResources(MlNativeDataFrameAnalyticsIntegTestCase.java:87)
10:00:37         at org.elasticsearch.xpack.ml.integration.MlNativeIntegTestCase.cleanUp(MlNativeIntegTestCase.java:158)
10:00:37         at org.elasticsearch.xpack.ml.integration.ClassificationIT.cleanup(ClassificationIT.java:90)
10:00:37 
10:00:37         Caused by:
10:00:37         java.lang.IllegalStateException: Timed out when waiting for persistent tasks after 30s
10:00:37             at org.elasticsearch.persistent.PersistentTasksService$2.onTimeout(PersistentTasksService.java:209)
10:00:37             at org.elasticsearch.cluster.ClusterStateObserver$ContextPreservingListener.onTimeout(ClusterStateObserver.java:325)
10:00:37             at org.elasticsearch.cluster.ClusterStateObserver$ObserverClusterStateListener.onTimeout(ClusterStateObserver.java:252)
10:00:37             at org.elasticsearch.cluster.service.ClusterApplierService$NotifyTimeout.run(ClusterApplierService.java:605)
10:00:37             at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:678)
10:00:37             at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
10:00:37             at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
10:00:37             at java.lang.Thread.run(Thread.java:748)
10:00:37 
10:00:37     java.lang.AssertionError: ClusterHealthResponse has timed out - returned: [{"cluster_name":"integTest","status":"red","timed_out":true,"number_of_nodes":2,"number_of_data_nodes":2,"active_primary_shards":12,"active_shards":18,"relocating_shards":0,"initializing_shards":0,"unassigned_shards":2,"delayed_unassigned_shards":0,"number_of_pending_tasks":0,"number_of_in_flight_fetch":0,"task_max_waiting_in_queue_millis":0,"active_shards_percent_as_number":90.0}]
10:00:37     Expected: is <false>
10:00:37          but: was <true>
10:00:37         at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
10:00:37         at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertNoTimeout(ElasticsearchAssertions.java:111)
10:00:37         at org.elasticsearch.test.ESIntegTestCase.ensureClusterSizeConsistency(ESIntegTestCase.java:1050)
10:00:37         at org.elasticsearch.test.ESIntegTestCase.afterInternal(ESIntegTestCase.java:575)
10:00:37         at org.elasticsearch.test.ESIntegTestCase.cleanUpCluster(ESIntegTestCase.java:2233)
10:00:37         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
10:00:37         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
10:00:37         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
10:00:37         at java.lang.reflect.Method.invoke(Method.java:498)
10:00:37         at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1750)
10:00:37         at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
10:00:37         at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
10:00:37         at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
10:00:37         at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
10:00:37         at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
10:00:37         at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)
10:00:37         at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)
10:00:37         at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:947)
10:00:37         at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:832)
10:00:37         at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:883)
10:00:37         at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:894)
10:00:37         at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
10:00:37         at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
10:00:37         at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
10:00:37         at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
10:00:37         at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
10:00:37         at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
10:00:37         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
10:00:37         at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
10:00:37         at java.lang.Thread.run(Thread.java:748)
```

--- a/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/ClassificationIT.java
+++ b/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/ClassificationIT.java
@@ -128,7 +128,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         return new NamedXContentRegistry(entries);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testSingleNumericFeatureAndMixedTrainingAndNonTrainingRows() throws Exception {
         initialize("classification_single_numeric_feature_and_mixed_data_set");
         String predictedClassField = KEYWORD_FIELD + "_prediction";
@@ -241,7 +240,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         assertEvaluation(KEYWORD_FIELD, KEYWORD_FIELD_VALUES, "ml." + predictedClassField);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testWithOnlyTrainingRowsAndTrainingPercentIsHundred() throws Exception {
         initialize("classification_only_training_data_and_training_percent_is_100");
         String predictedClassField = KEYWORD_FIELD + "_prediction";
@@ -479,7 +477,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
             "classification_training_percent_is_50_text_and_keyword", TEXT_FIELD + ".keyword", KEYWORD_FIELD_VALUES, "keyword");
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testWithOnlyTrainingRowsAndTrainingPercentIsFifty_DependentVariableIsBoolean() throws Exception {
         testWithOnlyTrainingRowsAndTrainingPercentIsFifty(
             "classification_training_percent_is_50_boolean", BOOLEAN_FIELD, BOOLEAN_FIELD_VALUES, "boolean");
@@ -524,7 +521,7 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
             }
         }
 
-        waitUntilAnalyticsIsStopped(jobId, TimeValue.timeValueMinutes(1));
+        waitUntilAnalyticsIsStopped(jobId);
 
         SearchResponse sourceData = client().prepareSearch(sourceIndex).setTrackTotalHits(true).setSize(1000).get();
         for (SearchHit hit : sourceData.getHits()) {
@@ -566,7 +563,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         assertThat(e.getMessage(), equalTo("Field [keyword-field] must have at most [30] distinct values but there were at least [31]"));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testDependentVariableCardinalityTooHighButWithQueryMakesItWithinRange() throws Exception {
         initialize("cardinality_too_high_with_query");
         indexData(sourceIndex, 6, 5, KEYWORD_FIELD);
@@ -623,7 +619,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         assertEvaluation(ALIAS_TO_KEYWORD_FIELD, KEYWORD_FIELD_VALUES, "ml." + predictedClassField);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testDependentVariableIsAliasToNested() throws Exception {
         initialize("dependent_variable_is_alias_to_nested");
         String predictedClassField = ALIAS_TO_NESTED_FIELD + "_prediction";
@@ -642,7 +637,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         assertEvaluation(ALIAS_TO_NESTED_FIELD, KEYWORD_FIELD_VALUES, "ml." + predictedClassField);
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testTwoJobsWithSameRandomizeSeedUseSameTrainingSet() throws Exception {
         String sourceIndex = "classification_two_jobs_with_same_randomize_seed_source";
         String dependentVariable = KEYWORD_FIELD;
@@ -687,7 +681,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         assertThat(secondRunTrainingRowsIds, equalTo(firstRunTrainingRowsIds));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testSetUpgradeMode_ExistingTaskGetsUnassigned() throws Exception {
         initialize("classification_set_upgrade_mode");
         indexData(sourceIndex, 300, 0, KEYWORD_FIELD);
@@ -785,7 +778,6 @@ public class ClassificationIT extends MlNativeDataFrameAnalyticsIntegTestCase {
         assertThat(stateIndexSearchResponse.getHits().getTotalHits().value, equalTo(0L));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/64926")
     public void testUpdateAnalytics() throws Exception {
         initialize("update_analytics_description");
 

--- a/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlNativeDataFrameAnalyticsIntegTestCase.java
+++ b/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlNativeDataFrameAnalyticsIntegTestCase.java
@@ -149,7 +149,7 @@ abstract class MlNativeDataFrameAnalyticsIntegTestCase extends MlNativeIntegTest
     }
 
     protected void waitUntilAnalyticsIsStopped(String id) throws Exception {
-        waitUntilAnalyticsIsStopped(id, TimeValue.timeValueSeconds(30));
+        waitUntilAnalyticsIsStopped(id, TimeValue.timeValueSeconds(60));
     }
 
     protected void waitUntilAnalyticsIsStopped(String id, TimeValue waitTime) throws Exception {

--- a/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java
+++ b/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java
@@ -330,7 +330,7 @@ public class RegressionIT extends MlNativeDataFrameAnalyticsIntegTestCase {
             }
         }
 
-        waitUntilAnalyticsIsStopped(jobId, TimeValue.timeValueMinutes(1));
+        waitUntilAnalyticsIsStopped(jobId);
 
         SearchResponse sourceData = client().prepareSearch(sourceIndex).setTrackTotalHits(true).setSize(1000).get();
         for (SearchHit hit : sourceData.getHits()) {
@@ -571,9 +571,7 @@ public class RegressionIT extends MlNativeDataFrameAnalyticsIntegTestCase {
 
         startAnalytics(jobId);
 
-        // This seems to some times take a bit longer than 30 seconds on CI
-        // so we give it a minute.
-        waitUntilAnalyticsIsStopped(jobId, TimeValue.timeValueMinutes(1));
+        waitUntilAnalyticsIsStopped(jobId);
 
         double predictionErrorSum = 0.0;
 

